test/CalculatorW3GTest.cpp

```diff
@@ -0,0 +1,144 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   CalculatorGtest.cpp
+ *  @date   Wed May 26 2021
+ *  @brief  Calculator Unit Tests
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include "ICalculatorW3.h"
+#include "SimpleExpressionCalculatorW3Factory.h"
+#include "AddCalculatorW3.h"
+#include "SubtractCalculatorW3.h"
+#include "MultiplyCalculatorW3.h"
+#include "DivideCalculatorW3.h"
+
+using namespace testing;
+using namespace calculator;
+
+/**
+ * @brief Fixture for Simple expression calulcators
+ * 
+ */
+class SimpleExpressionCalculatorW3Test : public ::testing::Test 
+{
+    protected:
+        AddCalculatorW3 addCalculator = AddCalculatorW3(10,10);
+        SubtractCalculatorW3 subtractCalculator = SubtractCalculatorW3(10, 10);
+        MultiplyCalculatorW3 multiplyCalculator = MultiplyCalculatorW3(10, 10);
+        DivideCalculatorW3 divideCalculator = DivideCalculatorW3(10, 10);
+};
+
+/**
+ * @brief fixtures for simple expression calculator factory
+ * 
+ */
+class SimpleExpressionCalculatorW3FactoryTest : public ::testing::Test
+{
+    protected:
+        SimpleExpressionCalculatorW3Factory simpleExpressionFactory;
+};
+
+/**
+ * @brief Test to make sure division by zero is handled correctly
+ * 
+ */
+TEST(CalculatorW3Tests, WhenDivideByZeroDivideCalculatorIsConstructed_ThenErrorPrintedToStdError)
+{
+    testing::internal::CaptureStderr();
+    DivideCalculatorW3 divideCalculator = DivideCalculatorW3(10, 0);
+    std::string output = testing::internal::GetCapturedStderr();
+    ASSERT_NE(output, std::string(""));
+}
+
+/**
+ * @brief Test to make sure a fully formed equation is formed when toString is called
+ * 
+ */
+TEST_F(SimpleExpressionCalculatorW3Test, WhenToStringIsCalledOnSimpleExpressionCalculators_ThenFullFormattedEquationIsReturned)
+{
+    ASSERT_TRUE(addCalculator.toString().find('=') != std::string::npos);
```

Comment:
Break each unit test for each operation.
Ensure each to string matches as we expected i..e
string expected = "5 + 5 = 10";
string actual = addCalculator.toString(); 
ASSERT_EQ(expected, actual)

This should be for all the cases listed above as well